repository/userRepository.py

```diff
@@ -0,0 +1,159 @@
+# w plikach typu repository, przechowuje sie metody i zapytania czysto odpowiedzialne za CRUD zapytan bazodanowych
```

Comment:
jak juz rozumiesz o co chodzi i dlaczego cos robimy to wywal te komentarze ze wszystkich plikow